### DISCUSSION TITLE
Not Work in Firefox 121.0 (64-bits)

### DISCUSSION THREAD

**Discussion by Ser4ph4** [Godiesc/firefox-gx#110] [General]
Hello, there is a bug in version 121.0 (64-bit)
from firefox...
![noname](https://github.com/Godiesc/firefox-gx/assets/78483475/87520c83-1d2a-435b-9a34-7f2e2b40d4d1)

**Comment by Godiesc** [comment]
hi, I assume you installed the version of the v.121 theme published yesterday?

**Reply by Ser4ph4** [reply]
Yes, that's right, is there a way to resolve this?

**Reply by Godiesc** [reply]
With solid theme:

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/1ba1a6cd-1e8e-4b3a-8632-57553c40b34b)

with theme with images:

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/e8a0a9a1-14d2-4f67-b385-6c68d049952e)

take care to read my notes of this new version of the theme to know the new problems that appear with this new version of the browser.

**Reply by Ser4ph4** [reply]
Ok, thanks, i changed theme its work now.
